engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -211,19 +213,40 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId,
   when(WorkflowExecutionAbortedState) {
     FSM.NullFunction
   }
+  when(WorkflowExecutionAbortingState) {
+    case Event(AbortedResponse(jobKey), stateData) =>
```

Comment:
As above, it's probably good to also catch a badly-timed success or failure here too